src/agent/rustjail/src/container.rs

```diff
@@ -473,11 +473,11 @@ fn do_init_child(cwfd: RawFd) -> Result<()> {
         if let Err(e) = sched::setns(fd, s) {
             if s == CloneFlags::CLONE_NEWUSER {
                 if e.as_errno().unwrap() != Errno::EINVAL {
-                    write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str());
+                    write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str())?;
                     return Err(e.into());
                 }
             } else {
-                write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str());
+                write_sync(cwfd, SYNC_FAILED, format!("{:?}", e).as_str())?;
```

Comment:
It's better not change the logic in a warning fix

Reply:
Why do you think it is safe to ignore the write error here?

If the `write_sync` result can safely be ignored, then maybe we need a `write_sync_ok` without a `Result` type that conveys the information in the type system and does the right thing. But I have presently no reason to believe it is OK to do that.

Reply:
We can do nothing with this error and I think this write_sync is redundant can be removed.